### PR TITLE
[12.0][MIG] purchase_analytic

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: purchase_analytic/views/purchase_views.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<openerp>
+    <data>
+        <record id="purchase_order_view_search_inherit_purchase_analytic" model="ir.ui.view">
+            <field name="name">purchase.order.search (purchase_analytic)</field>
+            <field name="model">purchase.order</field>
+            <field name="inherit_id" ref="purchase.view_purchase_order_filter"/>
+            <field name="arch" type="xml">
+                <data>
+                    <field name="create_uid" position="after">
+                        <field name="project_id" groups="purchase.group_analytic_accounting"/>
```
Comment (AaronHForgeFlow) [inline purchase_analytic/views/purchase_views.xml]: `purchase.group_analytic_accounting` does not exist

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]

**Review by AaronHForgeFlow — COMMENTED** [review]
It works well :muscle: Approving as soon as green.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: functional review

**Comment by AaronHForgeFlow** [comment]
@Cedric-Pigeon perhaps this is interesting to you.

**Review by Cedric-Pigeon — APPROVED** [review]
Code review. Thanks

**Pull request merged**